agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/init/SecondEntryPoint.java

```diff
@@ -258,6 +261,18 @@ public void customize(AutoConfigurationCustomizer autoConfiguration) {
               return props;
             })
         .addPropertiesCustomizer(new AiConfigCustomizer())
+        .addLogRecordProcessorCustomizer(
```

Comment:
can we add a comment to mention your issue https://github.com/open-telemetry/opentelemetry-java/issues/6599?